testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -121,6 +121,18 @@ def test_incompatible_n_workers(u):
     with pytest.raises(ValueError):
         FrameAnalysis(u).run(backend=backend, n_workers=3)
 
+
+def test_n_workers_conflict_raises_value_error(u):
+    analysis = FrameAnalysis(u.trajectory)
```

Comment:
I think you can just delete this line and/or use `analysis` object below instead of repeating the `FrameAnalysis` business.

Reply:
Ah yes my bad, I didn't notice that I still had it there, done :)